Formula/skaa.rb

```diff
@@ -0,0 +1,38 @@
+class Skaa < Formula
+  desc "Seven Kingdoms: Ancient Adversaries"
+  homepage "https://7kfans.com"
+  url "https://github.com/the3dfxdude/7kaa/releases/download/v2.15.5/7kaa-2.15.5.tar.xz"
+  sha256 "350a2681985feb4b71d20677d1a6d11921b9e3d97facbc94e4f020e848d8ab2b"
+  license "GPL-2.0-only"
+
+  depends_on "enet"
+  depends_on "gettext"
+
+  depends_on :macos
```

Comment:
This looks incorrect.

Reply:
@carlocab In what way? The audit tool told me I should put `depends_on :mac_os` after `enet` & `gettext`, but before `sdl`. 

By the way, the game can also be compiled for Linux, but I don't have a machine to test the formula on, hence I feel it's safer to just set `macOS` as dependency.

Reply:
Because the software doesn't depend on macOS. We should build it on Linux. If that build is broken, and someone else uses this on Linux, they can open a pull request to fix it. But they'll never get the chance to if we add the `:macos` dependency.